tools/create-release-archives.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Execute this script from the top-level directory (./tools/create-release-archives.sh)
+
+set -u
+
+remote_name=origin
+adapter_version="v1.2.3"
```

Comment:
I could imagine that we could even get both of these information from git, but let's don't overdo it right now..